flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/CheckpointCoordinatorTestingUtils.java

```diff
@@ -357,8 +359,11 @@ static ExecutionJobVertex mockExecutionJobVertex(
 		when(executionJobVertex.getParallelism()).thenReturn(parallelism);
 		when(executionJobVertex.getMaxParallelism()).thenReturn(maxParallelism);
 		when(executionJobVertex.isMaxParallelismConfigured()).thenReturn(true);
-		when(executionJobVertex.getOperatorIDs()).thenReturn(jobVertexIDs);
-		when(executionJobVertex.getUserDefinedOperatorIDs()).thenReturn(Arrays.asList(new OperatorID[jobVertexIDs.size()]));
+		List<OperatorIdPair> operatorIdPairs = new ArrayList<>();
+		for (OperatorID operatorID : jobVertexIDs) {
+			operatorIdPairs.add(new OperatorIdPair(operatorID, null));
```

Comment:
What do you think about a factory method to create `OperatorIdPair` from only generated ID?

Reply:
done, is it a terrible idea that both factory methods have the same name?

Reply:
I'd prefer more descriptive name for one-argument version.